OpenRA.Mods.Common/PlayerExtensions.cs

```diff
@@ -16,10 +16,9 @@ namespace OpenRA.Mods.Common
 {
 	public static class PlayerExtensions
 	{
-		public static bool HasNoRequiredUnits(this Player player)
+		public static bool HasNoRequiredUnits(this Player player, bool shortGame)
 		{
-			var mapOptions = player.World.WorldActor.Trait<MapOptions>();
-			if (mapOptions.ShortGame)
+			if (shortGame)
 				return !player.World.ActorsHavingTrait<MustBeDestroyed>(t => t.Info.RequiredForShortGame).Any(a => a.Owner == player);
 			return !player.World.ActorsHavingTrait<MustBeDestroyed>().Any(a => a.Owner == player && a.IsInWorld);
```

Comment:
For my own curiosity even though it's not actually in the PR: why is `a => a.Owner == player` sufficient for short game, but `a => a.Owner == player && a.IsInWorld` required for a long game?

Reply:
I noticed that myself after pushing the PR. *Probably* short game should have `&& a.IsInWorld` as well, but pinging @pchote in case this rings a bell somehow.

Reply:
Probably something to do with MCVs in transports...

Reply:
Ok. Since there aren't any tickets on this, I assume it's intentional and will leave it as-is.